guides/common/modules/proc_deleting-multiple-compliance-reports.adoc

```diff
@@ -4,7 +4,7 @@
 You can delete multiple compliance policies simultaneously.
 However, in the {ProjectWebUI}, compliance policies are paginated, so you must delete one page of reports at a time.
 ifdef::satellite[]
-If you want to delete all OpenSCAP reports, use the script in the {APIDocURL}chap-red_hat_satellite-api_guide-using_the_red_hat_satellite_api#sect-API_Guide-Deleting-OpenSCAP-Reports[Deleting OpenSCAP Reports] section of _{APIDocTitle}_.
+If you want to delete all OpenSCAP reports, use the script in the {APIDocURL}chap-red_hat_satellite-api_guide-using_the_red_hat_satellite_api#sect-API_Guide-Deleting-OpenSCAP-Reports[Deleting OpenSCAP Reports] section of the _{APIDocTitle}_.
```

Comment:
```suggestion
If you want to delete all OpenSCAP reports, use the script in {APIDocURL}chap-red_hat_satellite-api_guide-using_the_red_hat_satellite_api#sect-API_Guide-Deleting-OpenSCAP-Reports[Deleting OpenSCAP Reports] in _{APIDocTitle}_.
```

Reply:
The APIDoc is called "API Guide", so we should retain the "the" for now.